Fable.Remoting.Client/Proxy.fs

```diff
@@ -65,6 +68,7 @@ module Proxy =
                                             ContentType "application/json; charset=utf8"
                                             Cookie Fable.Import.Browser.document.cookie
                                             ]
+                   TokenCallback         = (fun _ -> emptyToken)
```

Comment:
Since we need to use some default `Promise` for the expression later on I thought I can simply use a default callback instead of `option`